charts/stable/thelounge/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: "4.2.0"
```

Comment:
Does this need to be quoted? Thought I saw we have it unquoted in other charts.

Reply:
I am not sure about this one. Perhaps @onedr0p or @bjw-s know better?

Reply:
You know, I think I had issues when using two digits for `appVersion`. Helm complained that it didn't want a float and so I had to put it in quotes. I don't think that this applies here.